src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Char.cs

```diff
@@ -408,6 +409,47 @@ public static unsafe int IndexOf(ref char searchSpace, char value, int length)
                     }
                 }
             }
+            else if (AdvSimd.IsSupported)
```

Comment:
Essentially the same comments as above.

Reply:
Sorry, didn't follow this comment. What is it that you are referring?

Reply:
I was saying that the comments applying to this method are essentially the same as applied to `SpanHelpers.Byte`. I just didn't want to retype them all out since the code is basically identical 😄

Reply:
Ah got it. :)